src/components/Formatters/useFileSizeFormatter.ts

```diff
@@ -0,0 +1,13 @@
+export const useFileSizeFormatter = () => {
+  const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB'];
+  const formatFileSize = (bytes: number): string => {
+    if (bytes === 0) {
+      return `${bytes} ${sizes[0]}`;
+    }
+    const sizeIndex = Number(Math.floor(Math.log(bytes) / Math.log(1024)));
+    return `${Math.round(bytes / Math.pow(1024, sizeIndex))} ${
+      sizes[sizeIndex]
+    }`;
+  };
+  return formatFileSize;
+};
```

Comment:
Correct, this could just be a regular function. I thought I had seen another thing like this that had been made as a hook, so I figured we were just doing that for semantic consistency. But it could totally be a function